spec/features/user_creates_account_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+require_relative '../factories/users'
+feature "user authentication" do
+# [ ] As a prospective user
+# I want to create an account
+# So that I can post items and review them
+#
+# [ ] As an unauthenticated user
+# I want to sign in
+# So that I can post items and review them
+#
+# [ ] As an authenticated user
+# I want to sign out
+# So that no one else can post items or reviews on my
+# behalf
+#
+# [ ] As an authenticated user
+# I want to update my information
+# So that I can keep my profile up to date
+#
+# [ ] As an authenticated user
+# I want to delete my account
+# So that my information is no longer retained by the app
+
+  describe "user creates an account" do
```

Comment:
Describe is for unit tests. Remove this line and just use a `let` statement for the `user` you need in both scenarios.